WordPress/src/main/java/org/wordpress/android/ui/FilteredRecyclerView.java

```diff
@@ -0,0 +1,327 @@
+package org.wordpress.android.ui;
+
+import android.content.Context;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.BaseAdapter;
+import android.widget.ProgressBar;
+import android.widget.RelativeLayout;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+import org.wordpress.android.models.FilterCriteria;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.DisplayUtils;
+import org.wordpress.android.util.NetworkUtils;
+import org.wordpress.android.util.helpers.SwipeToRefreshHelper;
+import org.wordpress.android.util.widgets.CustomSwipeRefreshLayout;
+import org.wordpress.android.widgets.RecyclerItemDecoration;
+
+
+public class FilteredRecyclerView extends RelativeLayout {
+
+    private ProgressBar mProgressLoadMore;
+    private SwipeToRefreshHelper mSwipeToRefreshHelper;
+    private Spinner mSpinner;
+    private boolean mSelectingRememeberedFilterOnCreate = false;
```

Comment:
Typo - should be `mSelectingRememberedFilterOnCreate` (drop the extra "e" from "Remem**e**bered")